vendor/assets/javascripts/jquery-ui-timepicker-addon.js

```diff
@@ -7,7 +7,13 @@
  * http://trentrichardson.com/Impromptu/MIT-LICENSE.txt
  */
 
-(function ($) {
+(function (factory) {
+	if (typeof define === 'function' && define.amd) {
+		define(['jquery', 'jquery-ui'], factory);
+	} else {
+		factory(jQuery);
+	}
+}(function ($) {
```

Comment:
This is interesting. So it looks like you've added some AMD module support? That's pretty neat.